src/loaders/mod_load.c

```diff
@@ -85,16 +89,22 @@ const struct mod_magic mod_magic[] = {
 	{"LARD", 1, TRACKER_UNKNOWN, 4},	/* in judgement_day_gvine.mod */
 	{"NSMS", 1, TRACKER_UNKNOWN, 4},	/* in Kingdom.mod */
 };
+#endif /* LIBXMP_CORE_PLAYER */
 
 static int mod_test(HIO_HANDLE *, char *, const int);
 static int mod_load(struct module_data *, HIO_HANDLE *, const int);
 
 const struct format_loader libxmp_loader_mod = {
+	#ifdef LIBXMP_CORE_PLAYER
```

Comment:
These should probably be non-indented for consistency with everywhere else? Several other locations too...

Reply:
Looked ugly without indented. If you feel strongly about it, I'll de-indent them.